crates/object_detection/src/pose_detection.rs

```diff
@@ -32,6 +29,8 @@ const DETECTION_IMAGE_HEIGHT: usize = 480;
 const DETECTION_IMAGE_WIDTH: usize = 192;
 const DETECTION_IMAGE_START_X: usize = (640 - DETECTION_IMAGE_WIDTH) / 2;
 
+const EXPECTED_OUTPUT_NAME: &str = "1525";
+
```

Comment:
Just a thought, maybe you can edit the xml by hand (technical term: model surgery :wink:) to remove the other outputs and also name this one properly

Reply:
Done